### PR TITLE
Course Outline

### PR DESCRIPTION
**Pull request by shiffman**
Here is my proposed course outline. I addressed #1 but smushing the introductory stuff into one session and having the thanksgiving week single session extend final project development. This isn't ideal but I figure I can catch up and continue with introductory material in the second week?

### CODE REVIEW

**Comment by shiffman** [comment]
Hi @ellennickles and @lydiajessup, in case you want to take a look! (Week 1 specific material coming soon.)

File: README.md [hunk 1]
```diff
@@ -1,28 +1,85 @@
...
-An introductory course designed to provide students with hands-on experience developing creative coding projects with machine learning. The history, theory, and application of machine learning algorithms and related datasets are explored in a laboratory context of experimentation and discussion. Examples and exercises will be demonstrated in JavaScript using the p5.js, ml5.js, and TensorFlow.js libraries. In addition, students will learn to work with open source pre-trained models in the cloud using Runway. Principles of data collection and ethics are introduced. Weekly assignments, team and independent projects, and project reports are required. 
+### 2: Machine Learning Models 1
+* Session 1 (M 9/9): Image Classification
+* Session 2 (W 9/11): Transfer learning
+
+### 3: Machine Learning Models 2
+* Session 1 (M 9/16): Pre-trained models
+* Session 2 (W 9/18): Physical Interaction
+
+### 4: Training a Model
```
Comment (joeyklee) [inline README.md:16]: Is this teachable machine / transfer learning? Or is this tf/ml5 neural network stuff?

File: README.md [hunk 2]
```diff
@@ -1,28 +1,85 @@
...
+### 11: Final Project Proposals
+* Session 1 (M 11/11): Group 1
+* Session 2 (M 11/13): Group 2
+
+### 12: Final Project Development
+* TBD (M 11/18)
+* TBD (W 11/20)
+* TBD (M 11/25)
+
+### 13: User Testing
```
Comment (joeyklee) [inline README.md:53]: The user testing week was excellent in the Nature of Code course. Definitely +💯 to this!

File: README.md [hunk 3]
```diff
@@ -1,28 +1,85 @@
...
-* Become familiar with the current landscape of new media art generated from machine learning algorithms. Understand how to use a machine learning model to generate media: words, sound, and images. 
+* Become familiar with the current landscape of new media art generated from machine learning algorithms. Understand how to use a machine learning model to generate media: words, sound, and images.
 
 ## EQUIPMENT
-You will need a modern laptop (4 years old or younger is a good rule of thumb).  Most required software is freely available. The department has all required commercial software installed on laptops available for checkout. 
+You will need a modern laptop (4 years old or younger is a good rule of thumb).  Most required software is freely available. The department has all required commercial software installed on laptops available for checkout.
 
 ## COURSE TEXTS and MATERIALS not provided
 
 [Deep Learning with JavaScript: Neural Networks in TensorFlow.js](https://amzn.to/2Z1Ie1L)
```
Comment (joeyklee) [inline README.md:80]: Note: that this book is not yet available. Only pre-orders for now.

File: README.md (hunk 1 above)
Comment (shiffman) [reply README.md:16]: I am tentatively planning to get to teachable machine and transfer learning in week 2. This week would be introducing some other pre-trained models like UNet / PoseNet / TBD. The "physical interaction" module is an idea proposed by @ellennickles to encourage students to create physical output (turn on an LED based on a gesture). So no new ML concepts just trying to move the application off the screen. I would LOVE help from anyone for the pcomp side of this! 😬

**Comment by shiffman** [comment]
Going to merge this so it's more easily visible, please open an issues or pull request with additional comments or questions!

**Pull request merged**